.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Faceted search CI
+
+on:
+  pull_request:
+    branches: [master, develop, main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build and test
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org/'
+          scope: '@talend'
+
+      - name: Install
+        run: yarn install --frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
I don't think we need it there as all faceted search dependencies are supposed to be public
```suggestion
```